src/models/AppState.js

```diff
@@ -254,6 +254,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
           return res._embedded.user.id;
         }
       },
+      'isIdxStateToken': {
+        deps: ['lastAuthResponse'],
+        fn: function (res) {
+          if (!res.stateToken) {
+            return null;
```

Comment:
should this return `false`?

Reply:
I think both would work, but i can change it to a boolean value to be consistent